### PR TITLE
use system creds

### LINKED ISSUE #13: For terraform we should prompt user for key OR create one
Currently for the terraform work the key is hardcoded

### PR DESCRIPTION
**Pull request by cooktheryan**
Fixes #13

### CODE REVIEW

**Comment by cooktheryan** [comment]
@sallyom can I get a review?

File: README.md [hunk 1]
```diff
@@ -151,40 +151,51 @@ cosign verify \
...
 ```
 
 Next, ensure that an image has been tagged with your quay repository and run the following. For this example, the image `quay.io/rcook/tools:awxy-runner2` is used.
 
 ```
-/usr/bin/cosign sign -y --fulcio-url=$FULCIO_URL --rekor-url=$REKOR_URL --oidc-issuer=$KEYCLOAK_OIDC_ISSUER  quay.io/rcook/tools:awxy-runner2
+cosign sign -y --fulcio-url=$FULCIO_URL --rekor-url=$REKOR_URL --oidc-issuer=$KEYCLOAK_OIDC_ISSUER  quay.io/rcook/tools:awxy-runner2
```
Comment (sallyom) [inline README.md:198]: if you set COSIGN_FULCIO_URL, COSIGN_REKOR_URL, COSIGN_OIDC_ISSUER, (and not sure if you need any others?) you won't need these flags! can be a follow-up though!

**Comment by cooktheryan** [comment]
adding for note purposes https://github.com/securesign/sigstore-ocp/blob/release-1.0.gamma/docs/sign-verify.md#signing-a-container-from-the-local-system

**Comment by sallyom** [comment]
LGTM! 😄

**Pull request merged**